views/userDetails.njk

```diff
@@ -5,14 +5,28 @@
 {%- from "govuk/components/tag/macro.njk" import govukTag -%}
 
  {% block beforeContent %}
+      {% if searchUrl === searchResultsUrl %}
+      {% set breadcrumbs = 
+        [
+          { text: "Home", href: homeUrl },
+          { text: "Manage user accounts", href: '/' },
+          { text: searchTitle, href: searchUrl },
+          { text: staff.name}
+        ]
+       %}
```

Comment:
need one less breadcrumb for the new search since results and filter are on the same page